sklearn/cluster/_birch.py

```diff
@@ -531,9 +531,11 @@ def _fit(self, X, partial):
 
         if self.copy != "deprecated" and first_call:
             warnings.warn(
-                "`copy` was deprecated in 1.6 and will be removed in 1.8 since it "
-                "has no effect internally. Simply leave this parameter to its default "
-                "value to avoid this warning.",
+                (
+                    "`copy` was deprecated in 1.6 and will be removed in 1.8 since it"
+                    " has no effect internally. Simply leave this parameter to its"
+                    " default value to avoid this warning."
+                ),
```

Comment:
Can you please remove those unwanted changes from this PR? They are impacting files unrelated to it.

I supposed they showed up when running the black formatter. It's a bit weird because if you used the same version of black as the one recommended on the comment in the PR they should not have changed other files.